specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -35,15 +35,15 @@ export const formDefinitionSpec = (table: SpecifyTable | undefined) =>
       syncers.xmlChild('enableRules', 'optional'),
       syncers.maybe(syncers.object(legacyBusinessRulesSpec()))
     ),
-    rows: rows(table),
+    definitions: definitions(table),
```

Comment:
As a reminder, while we use viewSpec for validation in the FormEditor, it's not yet used by the forms when rendering the forms - that will definitely change once visual form editor is available.
Until then, the change to the viewSpec only affects the form editor and form editor tests - that is why conditional form rendering code didn't have to be modified, despite this key being renamed